godot-codegen/src/util.rs

```diff
@@ -360,17 +360,16 @@ pub fn make_enum_definition(enum_: &Enum) -> TokenStream {
 
 pub fn make_constant_definition(constant: &ClassConstant) -> TokenStream {
     let name = ident(&constant.name);
-    let value = constant.to_constant();
-
-    if constant.name.starts_with("NOTIFICATION_") {
-        // Already exposed through enums
-        quote! {
-            pub(crate) const #name: i32 = #value;
-        }
+    let visibility = if constant.name.starts_with("NOTIFICATION_") {
+        quote!(pub(crate))
     } else {
-        quote! {
-            pub const #name: i32 = #value;
-        }
+        quote!(pub)
+    };
+
+    match constant.to_constant() {
+        NumType::I32(value) => quote!(#visibility const #name: i32 = #value;),
+        NumType::U32(value) => quote!(#visibility const #name: u32 = #value;),
+        NumType::I64(value) => quote!(#visibility const #name: i64 = #value;),
```

Comment:
Please use `quote! { ... }` syntax, with spaces in between, to follow existing style.

You can also shorten `visibility` to `vis`, as done elsewhere.

Reply:
done 😎